<p>
    Do not use <code>vavr.io.*</code> classes in public API. Try to keep the interfaces
    as generic as possible. Internal use of vavr classes is still possible.
</p>
<h2>Noncompliant Code Example</h2>
<pre>
interface MyIface {
    io.vavr.collection.List&lt;String> func1();
    void func2(io.vavr.collection.Map&lt;String,Object> param);
}
</pre>
<h2>Compliant Solution</h2>
<pre>
interface MyIface {
    java.util.List&lt;String> func1();
    void func2(java.util.Map&lt;String,Object) param);
}
</pre>